packages/eslint-plugin-wpcalypso/package.json

```diff
@@ -33,6 +33,6 @@
 		"@babel/core": "^7.16.0",
 		"@babel/eslint-parser": "^7.16.3",
 		"@babel/parser": "^7.16.4",
-		"eslint": ">=7.32.0"
+		"eslint": ">=8.2.0"
```

Comment:
1. I wonder why it's not updating eslint everywhere else in the repo
2. I also wonder why we need >=

Reply:
Looks like a copy/paste from the `peerDependency`. Some people like to have an open range for peer dependencies so we don't repeat the errors of react 17, where so many packages declared a peer dep on `react@^16` and they took so long to add `react@^17` to the mix.

But I don't think we should have those open ranges for `devDependencies` or regular `dependencies`.

Reply:
This is the PR for updating `eslint` in the rest of the repo: https://github.com/Automattic/wp-calypso/pull/56894

I have no idea why it open two different PRs.